### PR TITLE
Language frontend

### PR DESCRIPTION
**Pull request by cemalokten**
#### Issues this PR relates to #53 #73

- This PR is for the Front end parts of the issue, currently no functionality.
- Developed inside storybook
- Flags and languages have not been added
- Language search looks at both the language code and language name

### CODE REVIEW

File: client/src/components/Icon/icons/AR.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <rect y="0.429993" width="24" height="17.1429" fill="#128C4B" />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M5.71424 10.7157H14.8571V10.0801C14.8571 9.5845 15.3786 9.26217 15.8219 9.4838L16 9.57282V10.7157H17.619C17.9872 10.7157 18.2857 11.0142 18.2857 11.3823V11.6681C18.2857 12.4044 17.6887 13.0014 16.9523 13.0014H15.5238C15.1556 13.0014 14.8571 12.7029 14.8571 12.3347V11.8585H7.13324C6.95643 11.8585 6.78686 11.7883 6.66184 11.6633L5.71424 10.7157Z"
+        fill="white"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```
Comment (fadeomar) [inline client/src/components/Icon/icons/AR.js:30]: should be camelCase fillRule the same for all svg props

File: client/src/components/Icon/icons/AR.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
```
Comment (fadeomar) [inline client/src/components/Icon/icons/AR.js:42]: should be camelCase fillRule the same for all svg props

**Review by fadeomar — CHANGES_REQUESTED** [review]
hey @cemalokten 
  - [x] for icons file can please make sure that all props are in camelcase otherwise will throw errors  in console
  - [x] I'm able to run the code because you need to pull the latest develop branch with the new package json

**Review by fadeomar — CHANGES_REQUESTED** [review]
hey @cemalokten nice work: 
but there are some points that need to be fixed: 
  - [x] in the acceptability bar in Desktop https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=5099%3A26554 there is no icon for (Increase text size) 
  wireframe:  image
  storybook: 
image

  - [ ] also you can see the order is not correct
  storybook: 
image
 wireframe: 
image

  - [ ] the accessibility icon is not well exported (look to the man head ) 
  
image
image

  - [ ] important point is to add the accessibility bar to the pages or layout not only the storybook

**Comment by fadeomar** [comment]
hey @cemalokten to update you on this @ramy has fixed the general layout so you need only to add the accessibility bar in one place, pull the latest develop and you will see the change in the general layout 👍

**Comment by cemalokten** [comment]
1 - This has been updated so the magnifying icon is on both desktop and mobile - see https://www.figma.com/file/LK8egFhosvyvR6jp8yPC31/Hyde-Design-system?node-id=4601%3A23308&t=0DUpgJjY3dpNEo4S-0

2 - I've been following the order in the `Hyde Design System` file @jema28 which button order is correct?

image

**Review by fadeomar — APPROVED** [review]

File: client/src/components/Language/LanguageSelector/index.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  ar: 'Arabic',
+  en: 'English',
+  es: 'Spanish',
+  bm: 'Bambara',
+  as: 'French',
+  gs: 'Swedish',
+  vs: 'Japanese',
+  am: 'Bambara',
+  qm: 'German',
+  am: 'Russian',
```
Comment (fadeomar) [inline client/src/components/Language/LanguageSelector/index.js:25]: duplicate key 'am'

**Review by fadeomar — CHANGES_REQUESTED** [review]

**Comment by cemalokten** [comment]
@fadeomar Added missing flags and constants object for languages

**Review by fadeomar — APPROVED** [review]

**Pull request merged**